static/bidder-info/bwx.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://rtb.boldwin.live?pid={{.SourceId}}&host={{.Host}}&pbs=1"
+maintainer:
+  email: "prebid@bold-win.com"
```

Comment:
Prebid team has sent an email to verify specified maintainer's email address.  Requesting to reply back on email with "received" message

Reply:
email verified.